MLOps.NET/MLLifeCycleManager.cs

```diff
@@ -29,9 +31,21 @@ public async Task LogMetricAsync(Guid runId, string metricName, double metricVal
             await MetaDataStore.LogMetricAsync(runId, metricName, metricValue);
         }
 
+        /// <summary>
```

Comment:
I'm not against summary comments, but we should probably be consistent (meaning I should probably update the other methods too... :) 

With that said, this comment should probably be made more generic as what we are doing is just storing the model in a repository. In the future that can be other places than Azure, e.g. on a local fileshare

Reply:
Ohh yes,  have updated the comment to be more generic 👍